docs_src/core.ipynb

```diff
@@ -2598,6 +2598,106 @@
     "show_doc(even_mults)"
```

Comment:
<p>I think you mean `func_args` here, no?</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1710/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='124'/>